oper8/component.py

```diff
@@ -55,7 +55,7 @@ def __init__(
         """
         # Ensure that the name property is defined by accessing it and that
         # namespace is inherited from session.
-        self.name  # pylint: disable=pointless-statement
+        self.name  # noqa: B018
```

Comment:
This is very much a NIT/low ca but does ruff support long form disables? I always have issues trying to figure out what the shorthand means.

Reply:
Ah, yeah, TBH I don't know. I agree that the shorthand is much less readable, but I'm not aware of a long-form alias system

Reply:
Okay no worries! I saw there's a list of them in `pyproject.toml` so I think that's enough